client/src/legacy/HtmlEditorField.js

```diff
@@ -83,6 +83,7 @@ ss.editorWrappers.tinyMCE = (function() {
     getConfig: function() {
       var selector = "#" + editorID,
         config = jQuery(selector).data('config'),
+        height = config.row_height ? config.row_height : '400px',
```

Comment:
Where is that `400px` value coming from? Seems fairly arbitrary? I assume this is replacing what the deleted line below used to do?

Reply:
`400px` is the default value that we calculate on the server side if the setRows method was not applied. Yes, you're right, I'll change the logic here.

Reply:
Updated